x/evm/client/cli/native_tx.go

```diff
@@ -0,0 +1,47 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+	"github.com/spf13/cobra"
+)
+
+func NativeSendTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "native-send [from_key_or_address] [to_evm_address] [amount]",
+		Short: `Send funds from one account to an EVM address (e.g. 0x....).
```

Comment:
Is this only limited to sei account or from any account including evm address?

Reply:
this is specifically for sei address sending to evm address